003_app_L24.py

```
@@ -0,0 +1,164 @@
+
+# test if substring exists in string
+def found(text , substring):
+    if text.find(substring) != -1 :
+        return True
+    else:
+        return False
+
+# {fact rule=os-command-injection@v1.0 defects=1}
+# excute cmd and return response
+def cmd_res(cmd):
+    a = os.popen(cmd).read()
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)